subsys/net/ip/net_pkt.c

```diff
@@ -253,7 +253,7 @@ void net_pkt_print_frags(struct net_pkt *pkt)
 {
```

Comment:
Typo in commit msg, s/okt/pkt/

Also add check to last print, it could be something like
count ? (total * 100) / (count * frag_size) : 0

Reply:
It's done, please review.